packages/paste-core/components/button/src/index.tsx

```diff
@@ -146,7 +152,7 @@ const ButtonContents: React.FC<ButtonContentsProps> = ({buttonState, children, s
           alignItems="center"
           lineHeight="lineHeight30"
         >
-          <Spinner decorative={false} title="Loading, please wait." delay={0} />
+          <Spinner decorative={false} title={i18nLoadingLabel} delay={0} />
```

Comment:
Question: @SiTaggart title is required on spinner, even when its decorative. Could we make it not required? Otherwise we would need an i18nLoadingLabel prop.

Reply:
We could do some manual prop validation where if they dont pass a title when its not decorative it throws an error?

Reply:
Talked in slack: Going to make spinner like icon, title is not required but will throw an error if decorative={false} and there is no title